common/include/pcl/PCLPointCloud2.h

```diff
@@ -49,6 +51,51 @@ namespace pcl
   public:
     using Ptr = boost::shared_ptr< ::pcl::PCLPointCloud2>;
     using ConstPtr = boost::shared_ptr<const ::pcl::PCLPointCloud2>;
+
+    //////////////////////////////////////////////////////////////////////////
+    /** \brief Concatenate two pcl::PCLPointCloud2.
+      * \param[in|out] cloud1 the first input and output point cloud dataset
```

Comment:
The [correct syntax](http://www.doxygen.nl/manual/commands.html#cmdparam) is "[in,out]".